src/quo2/components/wallet/keypair/view.cljs

```diff
@@ -25,69 +25,78 @@
+                                 (colors/theme-colors colors/neutral-50 colors/neutral-40 theme))
+          :accessibility-label :options-button}]])]))
 
 (defn details-view
-  [address stored theme]
-  [rn/view
-   {:style {:flex-direction :row
-            :align-items    :center}}
-   [text/text
-    {:size                :paragraph-2
-     :accessibility-label :details
-     :style               {:color (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)}}
-    (details-string address stored)]
-   (when (= stored :on-keycard)
-     [rn/view {:style {:margin-left 4}}
-      [icon/icon :i/keycard-card
-       {:size  16
-        :color (colors/theme-colors colors/neutral-50 colors/neutral-40)}]])])
+  [details stored blur? theme]
+  (let [{:keys [address]} details]
+    [rn/view
+     {:style {:flex-direction :row
+              :align-items    :center}}
+     [text/text
+      {:size                :paragraph-2
+       :accessibility-label :details
+       :style               {:color (if blur?
+                                      colors/white-opa-40
+                                      (colors/theme-colors colors/neutral-50 colors/neutral-40 theme))}}
+      (details-string address stored)]
+     (when (= stored :on-keycard)
+       [rn/view {:style {:margin-left 4}}
+        [icon/icon :i/keycard-card
+         {:size  16
+          :color (if blur?
+                   colors/white-opa-40
+                   (colors/theme-colors colors/neutral-50 colors/neutral-40))}]])]))
 
 (defn- view-internal
   []
   (let [selected? (reagent/atom true)]
-    (fn [{:keys [theme accounts customization-color type details stored action on-options-press]}]
-      (let [{:keys [address full-name]} details]
-        [rn/pressable
-         {:style    (style/container @selected? customization-color theme)
-          :on-press #(when (= action :selector) (reset! selected? (not @selected?)))}
-         [rn/view {:style style/header-container}
-          [avatar type full-name customization-color]
-          [rn/view
-           {:style {:margin-left 8
-                    :flex        1}}
-           [title-view full-name action @selected? type customization-color on-options-press theme]
-           [details-view address stored theme]]]
-         [rn/flat-list
-          {:data      accounts
-           :render-fn account-list-card/view
-           :separator [rn/view {:style {:height 8}}]
-           :style     {:padding-horizontal 8}}]]))))
+    (fn [{:keys [accounts action] :as props}]
+      [rn/pressable
+       {:style    (style/container (merge props {:selected? @selected?}))
+        :on-press #(when (= action :selector) (reset! selected? (not @selected?)))}
+       [rn/view {:style style/header-container}
+        [avatar props]
+        [rn/view
+         {:style {:margin-left 8
+                  :flex        1}}
+         [title-view (merge props {:selected? @selected?})]
+         [details-view props]]]
```

Comment:
@OmarBasem - Since we pass the whole `props` to `details-view`. We need to de-structure the map in the `details-view` component.